src/elm/Pages/Build/History.elm

```diff
@@ -66,13 +52,36 @@ view now timezone page limit rm =
             text ""
 
 
+viewDeploymentBuilds : Posix -> Zone -> Org -> Repo -> Builds -> Html msg
+viewDeploymentBuilds now timezone org repo builds =
+    let
+        page : Page
+        page =
+            NotFound
+    in
+    if List.length builds > 0 then
+        div [ class "build-history" ]
```

Comment:
im in favor of the functionality, however the placement of this element within the deployment container is a bit random (just placed somewhere in the middle, and off-center if there are related builds in the widget).

the element also decreases the range of screen width before the containers start pushing other elements outside of the container.
(see a build with multiple related builds, and decrease the screen width)

its possible the current container doesnt support this widget to align nicely but im not sure placing it in the middle is my favorite option

Reply:
would it be possible to align the widget below the "Redeploy" link? something aligned to the container constraints should at least work nicer for smaller screens etc.

i dont have any other ideas at the moment but maybe someone else can chime in